src/library.rs

```diff
@@ -13,22 +16,29 @@ pub struct Library {
     pub(crate) inner: Arc<Inner>,
 }
 
-impl Default for Library {
-    fn default() -> Self {
+impl Library {
+    fn new(system: SystemCollectionData) -> Self {
         let mut user = CollectionData::default();
         user.is_user = true;
         Self {
             inner: Arc::new(Inner {
-                system: SystemCollectionData::Static(StaticCollection::new(
-                    &super::platform::STATIC_DATA,
-                )),
+                system,
                 user: Arc::new(RwLock::new(user)),
                 user_version: Arc::new(AtomicU64::new(0)),
             }),
         }
     }
 }
 
+#[cfg(any(target_os = "windows", target_os = "macos"))]
+impl Default for Library {
+    fn default() -> Self {
```

Comment:
this breaks `parley::FontCache` on linux (`context: fount::FontContext::new(&Library::default()),`)